File: DPBO-praktikum/4/cpp/Human.cpp
// memanggil header
#include "header.hh"

// constructor kosong
Human::Human(){

}

// constructor berparameter
Human::Human(string nik, string nama, string jenis_kelamin) {
    this->nik = nik;
    this->nama = nama;
    this->jenis_kelamin = jenis_kelamin;
}

// ambil nilai nik
string Human::get_nik() {
    return nik;
}

// mengisi nilai nik
void Human::set_nik(string nik) {
    this->nik = nik;
}

// ambil nilai nama
string Human::get_nama() {
    return nama;
}

// mengisi nilai nama
void Human::set_nama(string nama) {
    this->nama = nama;
}

// ambil nilai jenis kelamin
string Human::get_jenis_kelamin() {
    return jenis_kelamin;
}

// mengisi nilai jenis kelamin
void Human::set_jenis_kelamin(string jenis_kelamin) {
    this->jenis_kelamin = jenis_kelamin;
}

// destructor
Human::~Human(){

}

File: DPBO-praktikum/4/python/Human.py
class Human:
    def __init__(self, nik, nama, jenis_kelamin):
        self._nik = nik
        self._nama = nama
        self._jenis_kelamin = jenis_kelamin

    def get_nik(self):
        return self._nik

    def set_nik(self, nik):
        self._nik = nik

    def get_nama(self):
        return self._nama

    def set_nama(self, nama):
        self._nama = nama

    def get_jenis_kelamin(self):
        return self._jenis_kelamin

    def set_jenis_kelamin(self, jenis_kelamin):
        self._jenis_kelamin = jenis_kelamin